test/common/apiHelpers.js

```diff
@@ -167,6 +169,46 @@ function getBlocksToWaitPromise () {
 		});
 }
 
+function waitForConfirmations (transactions, limitHeight = 10) {
+
+	function checkConfirmations (transactions) {
+		return node.Promise.map(transactions, function (transaction) {
+			return getTransactionPromise(transaction);
+		})
+			.then(function (res) {
+				return node.Promise.each(res, function (result) {
+					node.expect(result).to.have.property('success').to.be.ok;
```

Comment:
Is `expect` throwing an error? I think the better idea is to throw it directly: `if (result.success === false) throw 'Not confirmed'`